exporters/webhook/plugins/pelorus_handler.py

```diff
@@ -35,10 +38,100 @@
     HTTPException,
     PelorusWebhookPlugin,
     PelorusWebhookResponse,
-    Request,
 )
 
 
+def _verify_payload_signature(
+    secret: bytes, signature_secret: str, json_payload_data: Dict[str, str]
+) -> bool:
+    """
+    This function attempts to match the hash of a payload to its data,
+    with the understanding that the input JSON may be formatted slightly
+    differently, such as having different separators or newlines.
+
+    Any change to a single character in the sender's JSON input will
+    change its hash signature. Therefore, on the webhook side,
+    this function attempts to ignore separators and newline characters
+    in order to match the incoming hash using variations of those formats
+    that do not affect the payload data.
+
+    To illustrate this point, consider the following variations of
+    the same JSON input, which are actually equivalent:
+
+    {"data": "value", "data2": "value2"}
+    {"data":"value","data2":"value2"}
+    {"data":"value", "data2":"value2"}
+    {"data" :"value","data2" :"value2"}
+    {"data" :"value", "data2" :"value2"}
+    {"data": "value", "data2": "value2"}\n
+    {"data":"value","data2":"value2"}\n
+    {"data":"value", "data2":"value2"}\n
+    {"data" :"value","data2" :"value2"}\n
+    {"data" :"value", "data2" :"value2"}\n
+    { "data": "value", "data2": "value2" }
+    { "data":"value","data2":"value2" }
+    { "data":"value", "data2":"value2" }
+    { "data" :"value","data2" :"value2" }
+    { "data" :"value", "data2" :"value2" }
+    { "data": "value", "data2": "value2" }\n
+    { "data":"value","data2":"value2" }\n
+    { "data":"value", "data2":"value2" }\n
+    { "data" :"value","data2" :"value2" }\n
+    { "data" :"value", "data2" :"value2" }\n
+
+    Returns:
+        bool: True when the matching hash was found, False otherwise
+    """
+
+    # Formats:
```

Comment:
is not this comment in the docstrings? lines 61-80

Reply:
Sort of, the comment contains all the variations, here for visibility only those that are within separator_formats, so maybe a bit of duplication, but makes the code readable.